nx-X11/programs/Xserver/os/utils.c

```diff
@@ -776,6 +777,11 @@ ProcessCommandLine(int argc, char *argv[])
 	    else
 		UseMsg();
 	}
+	else if ( strcmp( argv[i], "-co") == 0)
+	{
+	    if(++i >= argc)
+	        UseMsg();
```

Comment:
print a deprecation warning to stderr, if we get here... IMHO

Reply:
no, stderr output may cause irritations while ist should simply do nothing

Reply:
We're using calls such as  `fprintf(stderr, "Warning: Ignoring deprecated command line option '%s'.\n",` in nxagent's options parsing, so are you sure outputting to `stderr` would be bad idea here?